docs/UserCommands.md

```diff
@@ -0,0 +1,49 @@
+# User Command Reference
+
+`@rocket user`
+
+## Options to specify input
+
+* `edit`
+  * `--name` yourname
+  * `--email` emailaddress
+  * `--pos` yourposition
+  * `--github` yourrepo
+  * `--major` yourmajor
+  * `--bio` yourbio
+    * edit properties of your Launch Pad profile
+  * ADMIN or TEAM LEAD only option: `--member` MEMBER_ID
+    * edit properties of another user's Launch Pad profile
+
+* `view` MEMBER_ID
+  * view information about a user
+
+* `help`
+  * outputs options for user commands
+
+## ADMIN only
+
+* `delete` MEMBER_ID
+  * permanently delete member's Launch Pad profile
+
+* `admin`
+  * `--add` MEMBER_ID
+    * make existing user admin
+  * `--remove` MEMBER_ID
+    * remove admin rights from user
+
+* `lead`
+  * `--add` MEMBER_ID
+    * make existing user team lead
+  * `--remove` MEMBER_ID
+    * remove existing user as team lead
+
+## TEAM LEAD only
+
+* `add`
+  * `--team` TEAM_ID `--member` MEMBER_ID
+    * add user to team
+
+* `remove`
+  * `--team` TEAM_ID `--member` MEMBER_ID
+    * remove user from team
```

Comment:
I was thinking something similar, but then we'd need to take into account team leads as well - perhaps the best solution here is `@rocket permissions ...`? But in that case it's out of scope for this PR 😛

Reply:
Should we remove those options for now then?

Reply:
I know @bfbachmann doesn't like having an action and command in the same position but I don't mind - it makes sense to me and we do that for Inertia where both the following are commands:

```
inertia init
inertia <remote> up
```

never had anyone complain about it

Reply:
why not:

```
rocket user add <user> --lead
rocket user add <user> --admin
```

Reply:
We would also want to be able to change permissions on an existing user.

Reply:
oh wait in that case this is a bit different. Leads are team-level, whereas admins are org-level - I think admins should definitely have a separate set of commands `rocket admin ...`

I think it kind of makes sense to overload the `add` command to allow updates to user traits that are related to team - for example, `rocket user add <user> --lead` means "add <user> as a lead", which can apply even if they are already in team

Reply:
Another thoguht - now that i think of it, it makes more sense to have:

```
rocket team new
rocket team add <user1>
rocket team add <user2>
rocket team add <user1> --lead
```

Reply:
`rocket user remove` seems to imply removing a user from launch pad/ rocket

Reply:
I think there's a lot of confusion about how permissions are being handled that I have to clear up at the next meeting - I was planning to have distinct "admin" and "team lead" levels that are both org-wide, not team-dependent.

Reply:
Well I suppose the team lead level is org-wide, but when you create a lead you want to associate them with a team right?

Reply:
Like it doesnt make sense to create a lead without an associated team

Reply:
I wasn't planning to do that, because team leads can be independent of teams, e.g. when a new team starts. The idea being, a "team lead" is really just a name for an intermediate position between the exec team and "normal" members, and they have some team-related abilities (e.g. creating teams, editing teams) that aren't open to normal members.

Reply:
In any case, `add` and `remove` should definitely be moved to the `team` command.

Reply:
Ah, I see. Perhaps `admin` and `lead` both deserve their own subcommands then:

```
rocket lead set --team <team>
```

Reply:
I was considering having `@rocket permissions ...` to change permissions levels, and then adding/removing teams would be part of `@rocket team ...` (where it would check the user's permissions levels and return an error if the user wasn't at least a team lead).

Reply:
In any case we can discuss this in more detail next week - initially I think we can just remove the permissions- and teams-related commands from this PR, namely `add`, `remove`, `admin`, `lead` - sound good @jacqueschen1?